### ISSUE TITLE
[BUG] JSON not supported?

### ISSUE THREAD

**Issue by adhishthite** [microsoft/markitdown#34]
Based on the README, I see that JSON is a supported extension, however, when I passed a sample JSON to it, I get the following error:

```
markitdown._markitdown.UnsupportedFormatException: Could not convert '/tmp/SAMPLE-FILE.json' to Markdown.
The formats ['.json', '.json'] are not supported.
```

Please look into this, thanks!

**Comment by olin2011** [comment]
markitdown._markitdown.UnsupportedFormatException: Could not convert 'qa-1213.csv' to Markdown. The formats ['.csv'] are not supported.

I tried csv, json, and both had the same problem.

**Comment by realrajaryan** [comment]
After taking a look, I believe there was an assumption that **PlainTextConverter** class would’ve handled the file extensions mentioned under **“Various other text-based formats”** in the README. 

However, the PlainTextConverter class only handles files that **mimetypes** identifies as **text/***, but JSON text for example has a MIME type of **application/json**.

Adding converters for these extensions separately would be the best way to go about fixing this.

**Comment by Gad** [comment]
Would a converter that strips JSON text of its nested syntax delimiters be ok for the purpose of "indexing, text analysis" while keeping some indentation (thus loosing e.g. the distinction between key/values and arrays) or should reformatting it into e.g. tables  be more useful for this project ?

**Comment by Evgeny105** [comment]
I could be wrong, but in my case, the problem with text formats was solved when I installed the "mime-support" package additionally in the docker container

**Comment by afourney** [comment]
> After taking a look, I believe there was an assumption that **PlainTextConverter** class would’ve handled the file extensions mentioned under **“Various other text-based formats”** in the README.
> 
> However, the PlainTextConverter class only handles files that **mimetypes** identifies as **text/***, but JSON text for example has a MIME type of **application/json**.
> 
> Adding converters for these extensions separately would be the best way to go about fixing this.

You are correct. This was my assumption, and it was wrong. I'll address this asap.

**Comment by afourney** [comment]
Addressed in #261. Other PRs (#219 and #251) that do something more sophisticated are still under review.
